arduino/smartcar/smartcar.ino

```diff
@@ -58,8 +73,25 @@ int speed;
 const auto maxfrontUltDis = 100;
 SR04 frontUlt(arduinoRuntime, triggerPin, echoPin, maxfrontUltDis);
 
+const auto maxfrontUltDis1 = 100;
```

Comment:
Maybe add which side they're all facing instead of numbering them

Reply:
Maybe if thyre all the same distance you should add a constant and use instead.

Reply:
I fixed the four maxfrontUltDis so they reflect the direction front, back, left, and right. Also the reason I have fours because they might need to be changed in distance but if they don't later I'll remove it's only one remains